package/CHANGELOG

```diff
@@ -31,6 +31,8 @@ Changes
   * Maximum pinned versions in setup.py removed for python 3.6+ (PR #3139)
 
 Deprecations
+  * Deprecate structure of `results` attribute for LinearDensity, 
+    PersistenceLength and AnalysisFromFunction 
```

Comment:
Forgot, can you also add a reference to Issue 3115 here? Folks might benefit from looking at that discussion to understand the future plans are.

Reply:
Done!